code/modules/reagents/reagent_containers/glass_containers.dm

```diff
@@ -59,10 +59,10 @@
 	..()
 	if(is_open_container())
 		to_chat(usr, "<span class='notice'>You put the lid on [src].</span>")
-		flags ^= OPENCONTAINER
+		container_type ^= REFILLABLE | DRAINABLE
```

Comment:
so that the beaker don't become opaque when you put a lid on it

Reply:
Make sure is_open_container() will not remain `TRUE` with these, otherwise you can never take lids off.